src/api/oauth.ts

```diff
@@ -52,15 +52,15 @@ export class OAuth {
    * @returns The URL to authorize a user
    */
   static authorizeUrl(
-    { response_type = "code", redirect_uri, client_id, state, scope }: IAuthorizeUrlParams,
+    { response_type = "code", redirect_uri, client_id, state, scopes }: IAuthorizeUrlParams,
     client: AxiosInstance,
   ) {
     requireArgs({ client_id });
 
     const params = { response_type, client_id };
     if (redirect_uri) params["redirect_uri"] = redirect_uri;
     if (state) params["state"] = state;
-    if (scope) params["scope"] = scope;
+    if (scopes && scopes.length > 0) params["scope"] = scopes.join(" ");
```

Comment:
Should we join these with the "+" or "%20" delimiter?

Reply:
The + is added automatically. The results of the current implementation gives them a URL of: "https://test.com/oauth/authorize?response_type=code&client_id=6287ec36a841b25637c663de&state=1234567890&scope=assets:read+cms:write"
Which I think is what we want. I've added a test that validates this [here](https://github.com/webflow/js-webflow-api/pull/99/files#diff-09728765347cb5a95b3cde91ade8bd11d2d64f3f07a6a752c978080e0f46487cR126).